src/pages/staking/index.tsx

```diff
@@ -0,0 +1,703 @@
+  <Flex flexDir="column" alignItems="center" w="full" m="0 auto" {...props} />
+)
+
+const Divider = () => (
+  <Box
+    my={8}
+    w="10%"
+    height="0.25rem"
+    bgColor="homeDivider"
+    alignSelf="center"
+  />
+)
+
+const HeroStatsWrapper = (props: ChildOnlyProp) => (
+  <Flex
+    direction="column"
+    alignItems="center"
+    bg="layer2Gradient"
+    pb={8}
+    w="full"
+    {...props}
+  />
+)
+
+const ComparisonGrid = (props: ChildOnlyProp) => {
+  const gridAreas = {
+    base: `"solo-title"
+        "solo-rewards"
+        "solo-risks"
+        "solo-reqs"
+        "solo-cta"
+        "saas-title"
+        "saas-rewards"
+        "saas-risks"
+        "saas-reqs"
+        "saas-cta"
+        "pool-title"
+        "pool-rewards"
+        "pool-risks"
+        "pool-reqs"
+        "pool-cta";`,
+    xl: `"solo-title saas-title pool-title"
+        "solo-rewards saas-rewards pool-rewards"
+        "solo-risks saas-risks pool-risks"
+        "solo-reqs saas-reqs pool-reqs"
+        "solo-cta saas-cta pool-cta"`,
+  }
+  return (
+    <Grid
+      columnGap={12}
+      gridAutoRows="minmax(64px, auto)"
+      gridTemplateColumns={{ base: "1fr", xl: "repeat(3, 1fr)" }}
+      gridTemplateAreas={gridAreas}
+      sx={{
+        h4: {
+          color: "#787878",
+        },
+      }}
+      {...props}
+    ></Grid>
+  )
+}
+
+const H2 = (props) => (
+  <OldHeading
+    fontSize={{ base: "2xl", md: "2rem" }}
+    lineHeight={1.4}
+    mt={0}
+    {...props}
+  />
+)
+
+const ColorH3 = ({
+  color,
+  children,
+}: Pick<HeadingProps, "color" | "children">) => (
+  <OldHeading as="h3" fontSize="2xl" color={color}>
+    {children}
+  </OldHeading>
+)
+
+const StyledButtonLink = ({
+  href,
+  children,
+}: Pick<ButtonLinkProps, "href" | "children">) => (
+  <ButtonLink href={href}>{children}</ButtonLink>
+)
+
+const CardGrid = (props: ChildOnlyProp) => (
+  <Grid
+    gap={8}
+    templateColumns={{ base: "1fr", lg: "repeat(3, 1fr)" }}
+    {...props}
+  />
+)
+
+const StyledCard = (props: {
+  title: string
+  emoji: string
+  description: string
+  key: number
+  children: ReactNode
+}) => (
+  <Card
+    title={props.title}
+    emoji={props.emoji}
+    key={props.key}
+    description={props.description}
+    sx={{
+      justifyContent: "flex-start",
+      h3: {
+        fontWeight: "700",
+        margin: "0 0 1rem",
+      },
+    }}
+  >
+    {props.children}
+  </Card>
+)
+
+type BenefitsType = {
+  title: string
+  emoji: string
+  description: string
+  linkText?: string
+  to?: string
+}
+
+export const getStaticProps = (async (context) => {
+  const { locale } = context
+  const lastDeployDate = getLastDeployDate()
+
+  // load i18n required namespaces for the given page
+  const requiredNamespaces = getRequiredNamespacesForPage("/staking")
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      lastDeployDate,
+    },
+  }
+}) satisfies GetStaticProps<SSRConfig>
```

Comment:
`lastDeployDate` missing in prop types

Reply:
@pettinarip Any idea why this doesn't appear to be necessary? The deploy preview loads `lastDeployDate` for the footer on this page without any problem (which was why I left it off).
<img width="860" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/014b2921-e74d-44ca-9eef-58e8395c3a9f">

Did the same on the `deposit-contract`, `learn` and `roadmap/vision` pages as well without any type issues. If it's required for something I can add but wanted to note this.

Reply:
Not necessary. I just mentioned it for consistency with the rest.

So you see the types because it is inferred. That's fine, I'm good leaving this as it is.